app/Console/Commands/Egg/UpdateEggIndexCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Console\Commands\Egg;
+
+use Exception;
+use Illuminate\Console\Command;
+
+class UpdateEggIndexCommand extends Command
+{
+    protected $signature = 'p:egg:update-index';
+
+    public function handle(): int
+    {
+        try {
+            $data = file_get_contents('https://raw.githubusercontent.com/pelican-eggs/pelican-eggs.github.io/refs/heads/main/content/pelican.json');
+            $data = json_decode($data, true, 512, JSON_THROW_ON_ERROR);
+        } catch (Exception $exception) {
+            $this->error($exception->getMessage());
+
+            return 1;
+        }
```

Comment:
I think you need this?

```suggestion
        } catch (Exception $exception) {
            $this->error($exception->getMessage());
            
            report($exception);

            return 1;
        }
```